cfme/web_ui/__init__.py

```diff
@@ -3639,9 +3639,11 @@ class StatusBox(object):
 
     """
     def __init__(self, name):
-        self.name = name
+        assert type(name) == str
+        self.name = name.lower()
 
     def value(self):
-        return sel.element(
-            '//span[contains(@class, "card-pf-aggregate-status-count")]'
-            '/../../span[contains(., "{}")]/span'.format(self.name)).text
+        elem_text = sel.element(
+            "//span[contains(@class,'card-pf-aggregate-status-count')]"
+            "/../../../../../div[contains(@status, '{}')]".format(self.name)).text
```

Comment:
You see that I am refining the text() here: https://github.com/ManageIQ/integration_tests/pull/3338/files#diff-54c758ec2117f433d2c05581244ceb74R652 since if the element is not visible, then the text can't be read by selenium itself and it needs some help of JS, so that is why I recommended that :)